src/couchdb/rolling-database.ts

```diff
@@ -333,6 +350,37 @@ export function makeRollingDatabase<T>(
     )
   }
 
+  async function reduce<R>(
+    connection: ServerScope,
+    design: string,
+    view: string,
+    params: RollingReduceParams<R>
+  ): Promise<R[]> {
+    const {
+      afterDate,
+      cleaner,
+      partition,
+      useArchived,
+      // Native CouchDB options:
+      ...rest
+    } = params
+
+    const values = await rollingQuery<R>(
+      connection,
+      async db => {
+        const params: DocumentViewParams = { ...rest, reduce: true }
+        const response = await (partition == null
+          ? db.view(design, view, params)
+          : db.partitionedView(partition, design, view, params))
+        const [row] = response.rows
+        return row == null ? [] : [cleaner(row.value)]
+      },
+      { afterDate, useArchived }
+    )
```

Comment:
Does this only return the first document per database or can you pass a function to reduce over the rows returned by `rollingQuery`?